### PR TITLE
update: Get ATB param from site

### PR DESCRIPTION
**Pull request by mrshu**
* Load generated ATB param from DuckDuckGo.com site.

### CODE REVIEW

File: duckduckgo.safariextension/js/oninstall.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+*/
+"use strict";
+
+setTimeout(function() {
```
Comment (mrshu) [inline duckduckgo.safariextension/js/oninstall.js]: @bsstoner I had to put a timer here -- no other even was reliable enough. The value (500ms) is also quite deliberate, I am open to suggestions on better ways of doing this.

Comment (bsstoner) [inline duckduckgo.safariextension/js/oninstall.js]: We want this to run on the first page load after they install the extension right? Did you try adding a handler for DOMContentLoaded? 

```
document.addEventListener('DOMContentLoaded', function() {
  // check for atb here
}, false);
```

That might still not be enough though to get it to run after the internal code that also runs on that same event.

Comment (bsstoner) [inline duckduckgo.safariextension/js/oninstall.js]: Is there a way to wrap this so it only runs if `set_atb = false`?

**Comment by mrshu** [comment]
@bsstoner Thanks for your review -- I did it differently this time,  only a message to the background script is set at all times which should not be that computationally expensive.

**Comment by bsstoner** [comment]
👍  I think this is good to go. I had some initial issues testing, but I can't reproduce now and it seems okay.

**Comment by bsstoner** [comment]
👍

**Pull request merged**